### PR TITLE
Suggest closing other processes/tabs when Web USB gets "Unable to claim interface" error

### PR DESCRIPTION
**Pull request by microbit-grace**
https://microbit-global.monday.com/boards/1356069004/pulses/1356484978

### CODE REVIEW

**Comment by microbit-grace** [comment]
Screenshot 2024-01-26 at 11 25 08

File: src/messages/ui.en.json [hunk 1]
```diff
@@ -131,10 +131,11 @@
   "connectMB.tryAgain": "Try again",
 
   "connectMB.usbTryAgain.heading": "Connect using WebUSB",
-  "connectMB.usbTryAgain.subtitle": "You didn't select a micro:bit. Do you want to try again?",
-
-  "connectMB.usb.error1": "A WebUSB error occured.",
-  "connectMB.usb.error2": "Please unplug and replug the micro:bit before trying again.",
+  "connectMB.usbTryAgain.selectMicrobit": "You didn't select a micro:bit. Do you want to try again?",
+  "connectMB.usbTryAgain.closeTabs1": "Another process is connected to this device.",
+  "connectMB.usbTryAgain.closeTabs2": "Close any other tabs that may be using WebUSB (e.g. MakeCode, Python Editor, ML tool), or unplug and replug the micro:bit before trying again.",
```
Comment (microbit-grace) [inline src/messages/ui.en.json:136]: These were copied from python editor error https://github.com/microbit-foundation/python-editor-v3/blob/b2b1ca257f83ef17c54624e5aa3be8e972b0cc87/lang/ui.en.json#L1110-L1121 and tweaked to include `ML tool`. Not sure if that's correct... maybe ML tool is ambiguous to an outsider and it should be "machine learning tool"? Suggestions welcomed.

Comment (microbit-robert) [reply src/messages/ui.en.json:136]: I think we should go with "machine learning tool". I don't think we use "ML tool" in the app itself.

Comment (microbit-grace) [reply src/messages/ui.en.json]: 👍  https://github.com/microbit-foundation/ml-trainer/pull/193/commits/2bb412a9f7429ac660baebe036ae07d8be379c68

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**